build/images/ovs/Dockerfile.ubi

```diff
@@ -50,7 +50,7 @@ COPY charon-logging.conf /tmp
 COPY --from=ovs-rpms /tmp/ovs-rpms/* /tmp/ovs-rpms/
 RUN rm -f /etc/yum.repos.d/* && mv /tmp/CentOS.repo /etc/yum.repos.d/CentOS.repo && \
     curl https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official -o /etc/pki/rpm-gpg/RPM-GPG-KEY-centosofficial && \
-    subscription-manager config --rhsm.manage_repos=0 && \
+    sed -i.bak "s/^manage_repos = .$/manage_repos = 0/g" /etc/rhsm/rhsm.conf && \
```

Comment:
I found registry.access.redhat.com/ubi8:latest was updated 13 days ago. Is it the root cause of recent failure?

Reply:
It could be. I checked that `registry.access.redhat.com/ubi8:8.7-929` (current latest) reports this error, but the previous build `registry.access.redhat.com/ubi8:8.6-990` does not.

Reply:
Could we add a comment before this command to explain why we need to modify the file?